src/components/current-task/CurrentTaskTimer.jsx

```diff
@@ -113,6 +118,13 @@ export default function CurrentTaskTimer() {
             type="button"
             onClick={() => setPlay(!play)}
           />
+          <IconButton
+            className="current-task-timer__"
+            size="3rem"
+            icon="reset"
+            type="button"
+            onClick={() => reset()}
```

Comment:
I believe we can just go onClick={reset} if my memory doesn't fail me - the compiler is smart enough to understand it and just makes the code better to read.